pkgs/development/compilers/ghc/common-hadrian.nix

```diff
@@ -170,6 +171,11 @@
            then ./docs-sphinx-7-ghc98.patch
            else ./docs-sphinx-7.patch )
         ]
+        ++ lib.optional (lib.versionAtLeast version "9.6" && lib.versionOlder version "9.8")
+            (fetchpatch {
```

Comment:
Please use the `name` argument to fetchpatch to give this patch a descriptive name which helps when debugging.